frontend/src/pages/Login.vue

```diff
@@ -13,7 +13,7 @@
         </div>
       </b-container>
     </div>
-    <b-container class="mt--8">
+    <b-container v-if="!showPageMessage" class="mt--8 p-1">
```

Comment:
v-if="!showPageMessage" / v-else

I would change, so that there is no need for a negative query.

Reply:
same same 👆🏼